### ISSUE TITLE
Assassins Odyssey 1.5.1 Camera Breaks Final Patch

### ISSUE THREAD

**Issue by Dean-Gamer** [FransBouma/InjectableGenericCameraSystem#107]
The 1.5.1 version of the game has been released and is the final patch that will be released by Ubisoft!

It would be incredible if you could update the camera system to work with this version as a final update. 

The latest camera crashes the game, unfortunately.

Thank you so much for the amazon tool Frans, you have a great talent to be able to create this. I wouldn't know where to begin! 

I will send you a donation as soon as I can afford it :) 

Thanks again
Dean

**Comment by FransBouma** [comment]
Thanks :)

Strange tho, I have 1.5.1 and it works fine here. When exactly do you inject the camera? Do you have the steam version? I did hear about someone with the steam version which had a crash too but it did fix itself for some reason, will ask what they did.

**Comment by FransBouma** [comment]
From the user: 
> Regarding the earlier AC:Odyssey crash I got for the Steam version - it works now and it was because either I was playing in fullscreen or I was so dumb as to forget to disable the game's own photomode in Options.

Frankly I don't know why these would help, as it should work regardless, but as I have the Ubisoft version I don't know what's causing a crash. If you have the ubisoft version, then ignore this post ;)

**Comment by Dean-Gamer** [comment]
Thank you for your fast response!

I'm actually using the uPlay version, playing on 4k.

I did a bunch of tests injecting at the menu screen and when the game has already loaded with different display modes.

The only way I was able to get it working was by putting the game in windowed mode, injecting and then switching to full screen.

The only real issue is the resolution scaling of the camera menu in 4k as the ctrl zoom doesn't work with this method.

I am happy to have it working tho, even if it is with this strange workaround.
